atmo/templates/atmo/spark-job-results.html

```diff
@@ -0,0 +1,39 @@
+{% extends "atmo/base.html" %}
+
+{% block head_title %}Spark job results {{ job.identifier }}{% endblock %}
+
+{% block content %}
+<div class="page-header">
+    <h2>Spark Job results <small>{{ job }}</small></h2>
+</div>
+
+<h3>Job Artifacts</h3>
+<ul class="list-group">
+{% for item in data %}
+<li class="list-group-item">
+  {% if job.is_public %}
+    <a href="https://s3-{{ settings.AWS_CONFIG.AWS_REGION }}.amazonaws.com/{{settings.AWS_CONFIG.PUBLIC_DATA_BUCKET}}/{{ item }}">
+      {{ item }}
+    </a>
+  {% else %}
+    <samp>s3://{{settings.AWS_CONFIG.PRIVATE_DATA_BUCKET}}/{{ item }}</samp>
```

Comment:
Is this private URL used for AWS CLI? If yes, let's add a note outside the for loop how to use it, e.g. `aws s3 list ...`